target/linux/ath79/base-files/etc/hotplug.d/ieee80211/10_fix_wifi_mac

```diff
@@ -30,6 +30,14 @@ case "$board" in
 		[ "$PHYNBR" -eq 1 ] && \
 			echo $(k2t_get_mac "lan_mac") > /sys${DEVPATH}/macaddress
 		;;
+	trendnet,tew-823dru)
+	        # set the 2.4G interface mac address to LAN MAC
+	        [ "$PHYNBR" -eq 1 ] && \
+		        echo $(mtd_get_mac_text mac 4) > /sys${DEVPATH}/macaddress
```

Comment:
echo $(command) can be replaced with command, so
mtd_get_mac_text mac 4 > /sys${DEVPATH}/macaddress